placements/tests.py

```diff
@@ -68,6 +68,30 @@ def test_blog_order(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.data[0]['id'], str(pinnedEntry1.id))
         self.assertEqual(response.data[0]['pinned'], True)
+        # to test that the latest blog appears just below the pinned one
+        
+        BlogEntry.objects.all().delete()
```

Comment:
This can be moved to the test that requires blogs to be empty (that is, to `test_blog_order_without_pin`)